src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
```

Comment:
Use constants for the number of days threshold and the discount amount to avoid hardcoding values in the expression. This makes the code more readable and maintainable.